service/src/test/java/bio/terra/catalog/datarepo/DatarepoClientTest.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.catalog.datarepo;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import bio.terra.catalog.config.DatarepoConfiguration;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.client.auth.OAuth;
+import org.junit.jupiter.api.Test;
+
+class DatarepoClientTest {
+
+  private static final String BASE_PATH = "basepath";
+  private static final String TOKEN = "token";
+  private static final String AUTH_NAME = "googleoauth";
+
+  private final DatarepoClient client = new DatarepoClient(new DatarepoConfiguration(BASE_PATH));
+
+  @Test
+  void testApis() {
```

Comment:
would something like testApiTokens be more informative? This doesn't actually test any APIs

Reply:
It tests the API objects that the client code produces. The way it tests the objects is by looking at the auth tokens and the base path, but that testing method could change in the future, so it seems better to describe what it's doing rather than how it's doing it.

It could be written as a different test for each API method, but I wanted to add a test that verified that the http client was actually being shared, so it seemed easier to test all APIs in a single method.